front/app/containers/InitiativesShow/ReactionControl/Status/Answered.tsx

```diff
@@ -27,42 +21,15 @@ const StatusIcon = styled(Icon)`
   margin-bottom: 20px;
 `;
 
-const ReactionCounter = styled.div`
-  margin-top: 15px;
-  ${media.tablet`
-    display: none;
-  `}
-`;
-
-const ReactionText = styled.div`
-  display: flex;
-  justify-content: space-between;
-  align-items: flex-start;
-  padding-bottom: 4px;
-`;
-
-const ReactionTextLeft = styled.div`
-  font-size: ${fontSizes.base}px;
-  color: ${(props) => props.theme.colors.tenantPrimary};
-`;
-
-const ReactionTextRight = styled.div`
-  font-size: ${fontSizes.base}px;
-  color: ${(props) => props.theme.colors.tenantText};
-`;
```

Comment:
Not the case for this one, but I'm aware some of these statuses had different text colors than the component we're replacing it with. This will be looked at again in the final PR, where we do the polishing.